tests/person_spec.rb

```diff
@@ -0,0 +1,15 @@
+require '../classes/person'
+
+describe 'Create person object' do
+  context 'given the valid inputs' do
+    it 'should create a person with age = 10' do
+      person = Person.new(20)
+      expect(person.age).to eql(20)
+    end
+    it 'should create a person with age = 20 and called Omar' do
+      person = Person.new(20, 'Omar')
+      expect(person.age).to eql(20)
+      expect(person.name).to match('Omar')
+    end
+  end
+end
```

Comment:
- [ ] The person's class does have other methods defined in it. It will be nice to cover tests for them also.

See methods:
 - `can_use_services?`
 -  `validate_name`
 - `add_rental`
 -  `of_age?`

Reply:
I have added tests for the 3 methods except for `of_age?` because it's a private method.